action.yml

```diff
@@ -171,29 +140,57 @@ runs:
       env:
         GITHUB_TOKEN: ${{ inputs.pkg-token }}
         SONAR_TOKEN: ${{ inputs.sonar-token }}
-      run: ./gradlew sonarqube --info
       shell: bash
+      run: ./gradlew sonarqube --info
+
+
 
     - name: Build - Docker Image - echo
-      run: echo -e "🔔       D O C K E R      B U I L D"
       shell: bash
+      run: echo -e "🔔       D O C K E R      B U I L D"
 
     - name: Build - Docker Image
+      shell: bash
       run: |
         DOCKER_IMAGE="${{ inputs.docker-registry }}/${{ env.SERVICE_NAME }}:${{ inputs.docker-image-tag }}"
         docker build -t $DOCKER_IMAGE .
         echo "DOCKER_IMAGE=$DOCKER_IMAGE" >> $GITHUB_ENV
-      shell: bash
 
     - name: Release - push Docker image [on master]
       if: ${{ inputs.staging == 'true' }} || github.ref == 'refs/heads/master'
+      shell: bash
       run: docker push ${{ env.DOCKER_IMAGE }}
+
+
+
+    - name: Release - Terraform - echo
       shell: bash
+      run: echo -e "🔔       T E R R A F O R M"
+
+    - uses: dorny/paths-filter@v2
+      id: infrastructure-changes
+      with:
+        filters: |
+          terraform:
```

Comment:
this we should use parameter for terraform folder, and the default one is terraform

Reply:
Since Terraform is a given in this team we can make it a convention over configuration

Reply:
But sometimes we have a case when it is not matching with the terraform folder, like in vdfproxy.

But we probably don't have a single-repo-code which has this issue, so we can live with this for now

Reply:
Mhh I did move terraform scripts of vdfproxy into the /terraform folder ;)

Reply:
Good point. There we need to move the change detection into the new github action and apply it per change only